src/components/containers/ServiceForm/ServiceForm.js

```diff
@@ -37,16 +51,20 @@ export default class ServiceForm extends Component {
                     <TextField hintText="Enter href..." floatingLabelText="HREF"/><br></br>
                 </div>
                 <br></br>
-                <RaisedButton className="advanced" label="Advanced"/>
+                <RaisedButton className="advanced" label="Advanced" onClick={this.onClick.bind(this)}/>
                 <RaisedButton label="Submit" primary={true} />
 
-                <div>
-                    <TextField className='formtext'  hintText="" floatingLabelText="Category"/><br></br>
-                    <TextField className='formtext' hintText="" floatingLabelText="Description" multiLine={true} rows={2}/><br></br>
 
 
-                    <h6>End date</h6>
-                    <DatePicker className='formtext' hintText="End date" /><br></br>
+
+                {/**********************************************************.*********************************************/}
+                {/********************************************** ADVANCED FIELD IN FORM **********************************/}
+                {/**********************************************************.*********************************************/}
+            
+
+                <div style={ showAdvanced }>
```

Comment:
It is better to not render the advanced block at all than to hide it using inline-styling